types/deneb/block.yaml

```diff
@@ -0,0 +1,97 @@
+Deneb:
+  BeaconBlockBodyCommon:
+    # An abstract object to collect the common fields between the BeaconBlockBody and the BlindedBeaconBlockBody objects
+    type: object
+    description: "The [`BeaconBlockBody`](https://github.com/ethereum/consensus-specs/blob/master/specs/deneb/beacon-chain.md#beaconblockbody) object from the CL Deneb spec."
+    properties:
+      randao_reveal:
+        allOf:
+          - $ref: '../primitive.yaml#/Signature'
+          - description: "The RanDAO reveal value provided by the validator."
```

Comment:
nitpick: The spelling `RanDAO` is quite unconventional. It's usually `Randao` or `RANDAO`

Reply:
Updated in https://github.com/ethereum/beacon-APIs/pull/286/commits/e3dfda2125df409e1fe25fa3ef6299bd5d6a2728. Thanks!